src/main/java/be/syntra/devshop/DevshopBack/security/jwt/JWTTokenProvider.java

```diff
@@ -67,10 +67,10 @@ public Authentication getAuthentication(String token) {
                 Arrays.stream(claims.get(AUTHORITIES_KEY).toString().split(","))
                         .map(SimpleGrantedAuthority::new)
                         .collect(Collectors.toUnmodifiableList());
-        Long gebruikerId = null != claims.get(USER_ID)
+        Long userId = null != claims.get(USER_ID)
                 ? (Long) claims.get(USER_ID)
```

Comment:
deprecated. Will be changed as soon as DEV-008 is updated. This will do nicely until then :)